.github/workflows/update.yml

```diff
@@ -1,9 +1,10 @@
 name: 'Update'
 on:
   pull_request:
-  push:
-    branches-ignore:
```

Comment:
Why is this changed?

Reply:
I'm not sure why it was like this in the first place, but basically it resulted in us running CI double on each pull request, once for the branch trigger and once for the pull_request trigger. @goodlyrottenapple may know why we were doing this before, I suspect it was so that someone could debug this workflow in the past without opening a PR, but I don't know.

Either way, should not need to run it twice for every branch.